src/components/logs/app/index.html

```diff
@@ -0,0 +1,150 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Kubernetes logs</title>
+    <!-- meta http-equiv="Content-Security-Policy" -->
```

Comment:
~What does it mean for this to be empty - most restrictive or least restrictive?  The VS Code folks do like us to have a reasonably restrictive CSP if possible.~ Never mind - I see it gets replaced at runtime.